vllm/worker/cpu_model_runner.py

```diff
@@ -121,6 +145,16 @@ def _prepare_prompt(
             # is always the first token in the sequence.
             input_positions.extend(list(range(computed_len, seq_len)))
 
+            lora_id = seq_group_metadata.lora_int_id
+
+            if lora_id > 0:
+                lora_requests.add(seq_group_metadata.lora_request)
+
+            lora_index_mapping += [lora_id] * (seq_len - computed_len)
+            lora_prompt_mapping.extend([lora_id] * (
+                seq_len - computed_len if seq_group_metadata.sampling_params
+                and seq_group_metadata.sampling_params.prompt_logprobs else 1))
```

Comment:
Better to make these code inside a ```if self.lora_config``` scope? Seems these code may introduce additional overhead when LoRA is not used.